src/Cli/dotnet/commands/dotnet-test/TestApplication.cs

```diff
@@ -196,7 +196,7 @@ private Task<IResponse> OnRequest(IRequest request)
 
                     default:
                         // If it doesn't match any of the above, throw an exception
-                        throw new NotSupportedException(string.Format(LocalizableStrings.CmdUnsupportedMessageRequestTypeException, request.GetType()));
+                        throw new NotSupportedException(string.Format(Microsoft.DotNet.Tools.Test.LocalizableStrings.CmdUnsupportedMessageRequestTypeException, request.GetType()));
```

Comment:
Is this level of specificity really needed? Seemed like there are several instances late in this PR

Reply:
`LocalizableStrings` exists in multiple namespaces. The change for some of the files to use the correct Utils namespace made a conflict exist. This resolves the conflict.

In this specific case, `Microsoft.DotNet.Tools.Common` did not have `LocalizableStrings` but `Microsoft.DotNet.Cli.Utils` does. That's why there is a conflict now as that class also exists in `Microsoft.DotNet.Tools.Test`. The only other way is to create a using alias for this version of that class and use that alias here.